### PR TITLE
Firma 131 adapt every ui change in one place

### PR DESCRIPTION
**Pull request by SergioPH**

### CODE REVIEW

File: src/ui/cells/DetailCell.m [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+-(void)setCellValue:(NSString *)value
+{
+    self.valueLabel.text = value;
+}
+
+-(void)setDarkStyle
+{
+    UIFont *headerFont = [ UIFont fontWithName: @"Helvetica" size: 17.0 ];
```
Comment (ggdiez) [inline src/ui/cells/DetailCell.m]: Fonts and Colors must be in a specific style file

File: src/ui/cells/DetailCell.m [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    frameRect.size.width = 300;
+    self.titleLabel.frame = frameRect;
+    [self setTitleBoldStyle];
+}
+
+-(void)setClearStyle
+{
+    UIFont *titleFont = [ UIFont fontWithName: @"Helvetica" size: 16.0 ];
+    self.titleLabel.font = titleFont;
+    self.titleLabel.textColor = [UIColor colorWithRed:160.0f/255.0f
```
Comment (ggdiez) [inline src/ui/cells/DetailCell.m]: Fonts and Colors must be in a specific style file

File: src/ui/cells/DetailCell.m (hunk 1 above)
Comment (SergioPH) [reply src/ui/cells/DetailCell.m]: Done

File: src/ui/cells/DetailCell.m (hunk 2 above)
Comment (SergioPH) [reply src/ui/cells/DetailCell.m]: Done

**Pull request merged**